onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -276,6 +275,13 @@ template <typename T>
 Attention<T>::Attention(const OpKernelInfo& info) : OpKernel(info), AttentionCPUBase(info) {
 }
 
+template <typename T>
+void Attention<T>::FreePackedWeights() {
```

Comment:
reset is_prepack_ as well?

Reply:
its just extra statements that doesn't add any value. is_prepack_ is set to true only once when PrePack() is successful, there is no other fn that can change this. 

In fact, this fn is necessary only because of the amount of memory weights can use.